atlp-rwanda/Team-furebo-E-commerce-fn
# ft-two-factor-authentication-for-sellers-185172182

#### What does this PR do?
This PR implements two-factor authentication for sellers.
#### Description of Task to be completed?
The task involves adding two-factor authentication functionality specifically for sellers. Sellers will be required to provide  a verification code, in addition to their username and password when logging into their accounts.
#### How should this be manually tested?
1. Log in to the seller account using valid credentials.
2. Navigate to the user profile page.
3. Enable two-factor authentication.
4. Attempt to log in again using the same credentials.
5. Enter the verification code obtained from in the verification page.
6. Verify that the login is successful.
7. Navigate to the user profile and disable two-factor authentication

#### What are the relevant pivotal tracker/Trello stories?
#185172182
